ops/staging-deploy.tmpl.yaml

```diff
@@ -86,19 +86,19 @@ extraEnvVars: &envVars
     value: http://fcrepo.staging-fcrepo.svc.cluster.local:8080/rest
   - name: GOOGLE_FONTS_KEY
     value: $GOOGLE_FONTS_KEY
-  # - name: GOOGLE_ANALYTICS_ID
-  #   value: $GOOGLE_ANALYTICS_ID  # Set in GitHub Environment Variables Section
-  # - name: GOOGLE_OAUTH_APP_NAME
+  # - name: ANALYTICS_ID
```

Comment:
I don't feel like I understand this change?

Reply:
@orangewolf the idea was that since this code is going back into hyku, to not limit the person to using google analytics. even though that is what pals is using. so instead of prefacing everything with "google", we leave it more open.

the change in this file specifically is to match what we have in "account_settings.rb" and everywhere else now.

Reply:
although they will all get removed too on #482